src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int AMOUNT_OF_FIGURES = 6;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure [] figures = new Figure[AMOUNT_OF_FIGURES];
+        StringBuilder builder = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder result = new StringBuilder();
```
Let's add context to the var name

Reply:
Done